### PR TITLE
Increase size of root volume

### PR DESCRIPTION
**Pull request by mccalluc**

### CODE REVIEW

File: deployment/stack.py [hunk 1]
```diff
@@ -191,6 +191,15 @@ def make_template(config, config_yaml):
             'Monitoring': True,
             'SecurityGroups': [functions.ref("InstanceSecurityGroup")],
             'Tags': instance_tags,
+            'BlockDeviceMappings': [
+                {
+                    'DeviceName': '/dev/sda1',
+                    'Ebs': {
+                        'VolumeSize': '10.5',
```
Comment (hackdna) [inline deployment/stack.py]: [VolumeSize must be an integer](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-blockdev-template.html#cfn-ec2-blockdev-template-volumesize).

**Comment by hackdna** [comment]
"For valid values, see the Size parameter for the [CreateVolume](http://docs.aws.amazon.com/AWSEC2/latest/APIReference/ApiReference-query-CreateVolume.html) action in the Amazon EC2 API Reference."

**Comment by scottx611x** [comment]
@hackdna Yes, but the confusing part is that its a string in the example shown at the first link.
```
{
   "DeviceName":"/dev/sdc",
   "Ebs":{
      "SnapshotId":"snap-xxxxxx",
      "VolumeSize":"50",
      "VolumeType":"io1",
      "Iops":"1000",
      "DeleteOnTermination":"false"
   }
}     
```

**Comment by mccalluc** [comment]
flake 8 failure: There is a bug to look into why this doesn't match our precommit hook

**Comment by scottx611x** [comment]
@mccalluc regarding flake8 failure: [This issue](https://github.com/refinery-platform/refinery-platform/issues/1851) was closed but may be worth reopening now?

**Comment by mccalluc** [comment]
Stack is up, and disk sizing looks good:
```
$ df -h /
Filesystem      Size  Used Avail Use% Mounted on
/dev/xvda1       11G  4.3G  6.0G  42% /
```

Next, want to install a tool, upload a dataset, and make sure visualizations work.

**Comment by hackdna** [comment]
Great, now lets make sure the expanded space is sufficient when Docker functionality is used.

**Comment by mccalluc** [comment]
And tool install and launch went smoothly (IGV + HiGlass + debug tool), and we still have some free space:
```
$ df -h /
Filesystem      Size  Used Avail Use% Mounted on
/dev/xvda1       11G  6.5G  3.7G  64% /
```
Merging

**Pull request merged**